templates/project_stl_containers/NewDetector/NewDetectorPoint.h

```diff
@@ -27,6 +27,7 @@ class NewDetectorPoint
      *@param eLoss    Energy deposit [GeV]
      **/
     NewDetectorPoint(Int_t trackID,
+                     uint32_t evtId,
```

Comment:
@kresan  Can you verify if this is a meaningful/correct/intended way of passing/initializing the event id (why is this difference between the stl and root container templates?)

Reply:
EventID is not needed in the NewDetectorPoint class, please remove it from both templates. Ideally, project_stl_containers should not use ROOT types.

Reply:
Removed `fEventId` and all users from `project_stl_containers`. Please recheck now.